R/description.R

```diff
@@ -51,63 +51,59 @@
 #'
 #' use_description_defaults()
 #' }
-use_description <- function(fields = NULL, check_name = TRUE) {
+use_description <- function(fields = list(), check_name = TRUE) {
   name <- project_name()
   if (check_name) {
     check_package_name(name)
   }
-  fields <- fields %||% list()
-  check_is_named_list(fields)
-  fields[["Package"]] <- name
 
-  desc <- build_description(fields)
-  desc <- desc::description$new(text = desc)
-  tidy_desc(desc)
+  desc <- build_description(fields, package = name)
   lines <- desc$str(by_field = TRUE, normalize = FALSE, mode = "file")
 
   write_over(proj_path("DESCRIPTION"), lines)
-
   if (!getOption("usethis.quiet", default = FALSE)) {
     print(desc)
   }
 }
 
 #' @rdname use_description
+#' @param package Package name
 #' @export
-use_description_defaults <- function() {
-  list(
-    usethis.description = getOption("usethis.description"),
-    devtools.desc = getOption("devtools.desc"),
-    usethis = list(
-      Package = "valid.package.name.goes.here",
-      Version = "0.0.0.9000",
-      Title = "What the Package Does (One Line, Title Case)",
-      Description = "What the package does (one paragraph).",
-      "Authors@R" = 'person("First", "Last", , "first.last@example.com", c("aut", "cre"), comment = c(ORCID = "YOUR-ORCID-ID"))',
-      License = " `use_mit_license()`, `use_gpl3_license()` or friends to pick a license",
-      Encoding = "UTF-8",
-      LazyData = "true",
-      Roxygen = "list(markdown = TRUE)"
-    )
```

Comment:
I had structured it this way so people could see the defaults without reading usethis source, by calling this functions. But I think we've talked about this before and you don't find that compelling.

Reply:
Yeah, it just felt like too many layers too me, and you couldn't easily see what would actually get used, which also seems like a bummer.